### ISSUE TITLE
Very high QUAL values

### ISSUE THREAD

**Issue by mbhall88** [freebayes/freebayes#782]
Hi,

I was wondering if you might be able to explain some very high QUAL scores in my VCFs?

I am calling variants from recent, high-quality bacterial (i.e., haploid) nanopore data.

This is the command I am using

```
freebayes-parallel <(fasta_generate_regions.py "$ref" 500000) $threads \
    --haplotype-length -1 -m 10 -q 10 -p 1 --min-coverage 2 -f "$ref" "$aln"
```

As an example, this is the distribution of QUAL values for one sample

![image](https://github.com/freebayes/freebayes/assets/20403931/9516e440-5bc0-429d-b0a7-a65be0ef1f7f)

and here are some quality scores for the same sample using different variant callers (I appreciate they aren't necesarily the same as QUAL will be calculated based on different criteria)

#### DeepVariant

![image](https://github.com/freebayes/freebayes/assets/20403931/c5a9c0c0-6ed5-4b87-b11b-42ff88a62ee8)

#### bcftools

![image](https://github.com/freebayes/freebayes/assets/20403931/ebdc4923-33b6-4f4a-9126-7bca509e0062)

---

I'm wondering if there are some different parameters you think I could use to try and bring these QUAL scores down into a range similar to other callers? I want to benchmark ROC curves and do that I need to filter all VCFs on something, and that something is normally QUAL, but it makes it very hard to do this as if I filter 0-100 with a step increment of 5, most of these freebayes calls will be included still.

**Comment by mbhall88** [comment]
Commenting to keep this open. Do any maintainers have any comment on this?

**Comment by mbhall88** [comment]
I guess there is no comment on this? I will let it lapse after the next reminder.

**Comment by PMuchina** [comment]
Was this resolved? I am facing the same issue. It is hard to filter the VCF based on QUAL.

**Comment by mbhall88** [comment]
No, it hasn't been. If, like me, you're using ONT data, [my recommendation would be Clair3](https://elifesciences.org/articles/98300).
